ext/AdvancedHMCMCMCChainsExt.jl

```diff
@@ -15,21 +15,21 @@ function AbstractMCMC.bundle_samples(
     sampler::AbstractMCMC.AbstractSampler,
     state,
     chain_type::Type{Chains};
-    discard_initial = 0,
-    thinning = 1,
-    param_names = missing,
-    bijector = identity,
+    discard_initial=0,
+    thinning=1,
```

Comment:
@ErikQQY, consider avoiding these styling changes in PRs.  It's okay to do it via a separate PR if we decide to follow a different style.

Reply:
I think autoformatting would make these same changes if we copied the `.JuliaFormatter.toml` from here: https://github.com/TuringLang/Turing.jl/blob/main/.JuliaFormatter.toml

I think that would be a good PR to make, but I agree that it should be a separate PR, because it's hard and a bit risky to read diffs that mix actual changes made by a human with autoformatting changes in unrelated parts of the codebase.

Reply:
Yeah, I was trying to format before push habitually but found out the formatting style has not been configured yet.